staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -490,6 +490,14 @@ type KubeletConfiguration struct {
 	// Default: "/etc/resolv.conf"
 	// +optional
 	ResolverConfig string `json:"resolvConf,omitempty"`
+	// TODO(mtaufen): It is increasingly looking like nobody actually uses the
+	//                Kubelet's runonce mode anymore, so it may be a candidate
+	//                for deprecation and removal.
+	// If runOnce is true, the Kubelet will check the API server once for pods,
+	// run those in addition to the pods specified by static pod files, and exit.
+	// Default: false
+	// +optional
+	RunOnce bool
```

Comment:
I think the test you mentioned on Slack may be failing because this doesn't have the `json:"runOnce,omitempty"` tag.